data-collection/src/main/java/refactoringml/db/Yes.java

```diff
@@ -1,34 +1,53 @@
 package refactoringml.db;
 
+import refactoringml.util.JGitUtils;
 import refactoringml.util.RefactoringUtils;
 
 import javax.persistence.*;
 import java.util.Calendar;
 
+//TODO: create a Baseclass for both Yes and No, as they share a lot of logic
 @Entity
-@Table(name = "yes", indexes = {@Index(columnList = "project_id"), @Index(columnList = "refactoringType"), @Index(columnList = "refactoring"), @Index(columnList = "isTest")})
+@Table(name = "yes", indexes = {@Index(columnList = "project_id"), @Index(columnList = "refactoringLevel"), @Index(columnList = "refactoring"), @Index(columnList = "refactoringSummary"), @Index(columnList = "isTest")})
```

Comment:
Refactoring summary is not a good index, I'd say, as it will have a lot of variability, and we won't query it, I think.

Reply:
ok, good point